lib/ansibledocgen/cli.py

```diff
@@ -22,11 +22,18 @@ def __init__(self):
                           Default is markup. Example: --style=[markup]",
                           metavar="STYLE",
                           default="markup")
+        parser.add_option("-n", "--no-tags", dest="show_tags",
+                          action='store_false',
+                          help="This opcion disable show tags in documentation",
```

Comment:
I would change this to help="This option disables show tags in the documentation"